app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,56 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        From the home screen, swipe down to display the favorites panel and swipe up to hide it.
+        Swipe down from the home screen to display the favorites panel, and upwards to hide it.
         \n
-        \nIf you swipe up when the favorites panel is hidden, it will display the complete list of applications.
-        Swipe down from the top of this list to come back to the home screen.
+        \nSwipe up to show all apps if the favorites panel is hidden.
+
+        Swipe down from the top of it to return to the home screen.
         \n
-        \nPress the button below the favorites panel to display the menu of the launcher.
+        \nThe button below the favorites panel opens the launcher menu.
         \n
-        \nIf you hold your finger a few moments on an application or a folder, you will be offered to access its settings.
+        \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
-        The notification allows to display a popup with favorites above other applications when you leave the home screen.
-        In most cases, it should work directly and without issues.
+        Shows a popup of favorites above other apps when you leave the home screen.
         \n
-        \nHowever, in some special cases it may bring back to the home screen instead of being displayed over other applications.
-        \nIf you are facing this issue, there are two things you can try:
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
+        \If instead it brings you back to the home screen:
+        \n- Press the \"clear all\" button in \"Recent Apps\" or
+        \n- Reboot your device
         \n
-        \nTo not be intrusive, the notification icon is transparent and low priority.
-        With some applications having even lower priorities, this may result in an empty space in the status bar.
+        \nUnintrusively, the notification icon is transparent and of low priority.
+        With some apps having even lower priorities, this may result in empty space in the status bar.

```

Comment:
It isn't obvious how priorities change visibility.

Reply:
Yes, I struggle a bit to explain this one...

Notification icons in the status bar are ordered based on several criteria, including the priority.
For example, if I put the maximum priority, the (transparent) icon would be displayed first and there would be an empty space at the beginning of the status bar.

To avoid that as much as possible, I assigned a low priority to the notification so it is rather displayed at the end of the bar (there is still an invisible space but it doesn't create a hole anymore among other icons).
However, if several apps have low priority, it is not clear how their notifications will be ordered and the empty space may appear again (this is what the last sentence is trying to say).

Reply:
```suggestion
        If other apps are prioritized lower, it can still happen.

```

Reply:
```suggestion
        If other apps are prioritized lower, it can still happen.
```
We can remove the "all" because if not all are prioritized lower, the empty space will be somewhere in the middle of the notifications icons rather than at the very beginning.
=> I will adjust my comment about the previous sentence to put "in the status bar" as it cover also these cases.

Reply:
I tried changing my proposal, but it doesn't account for it moving around.
Never noticed or think of it as a big issue. Having a lot of text (that isn't even good) in the UI is a much bigger issue IMO. Can it be removed and tracked as an issue? Maybe there is a possible technical fix?